### ISSUE TITLE
For local Hostnames all logins will be provided regardless of subdomains

### ISSUE THREAD

**Issue by msv1** [keepassxreboot/keepassxc-browser#1223]
![urls](https://user-images.githubusercontent.com/43469827/108196061-ee885d00-7118-11eb-9efa-8bf88e81343b.png)
![bare](https://user-images.githubusercontent.com/43469827/108196066-ef20f380-7118-11eb-8a0f-36bbe0053e16.png)

When I click on a loginfield in a loginpage of a local domain I get all logins for this Hostname and not only for this "subdomain" (urls.png)
## Expected Behavior
I would like to receive only the login data for this specific page

## Current Behavior
I get all logins for my local Hostname (see example: bare.png)

## Steps to Reproduce (for bugs)
create different subpages with logins beyond a local Hostname

## Debug info

KeePassXC - 2.6.4
KeePassXC-Browser - 1.7.6
Operating system: Linux x86_64
Browser: Mozilla Firefox 85.0

**Comment by droidmonkey** [comment]
Did you enable "Only return best matching results? " in the browser settings in keepassxc?

**Comment by msv1** [comment]
Yes, sure. Its working well for "normal" Internet domains like _"www.name.com"_

**Comment by varjolintu** [comment]
With the latest KeePassXC you can use an exact URL inside `"` characters. This will solve your problem.
